assets/models/system/salesforce-xgen-7b-4k-base/spec.yaml

```diff
@@ -0,0 +1,20 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: salesforce-xgen-7b-4k-base
+path: ./
+properties:
+  SHA: 7dde7ac26ddcb679e0429fed1b6f751c1ffab1fb
+  datasets: databricks/databricks-dolly-15k
```

Comment:
need to change

Reply:
Please cross verify all tags and properties. Take help from inference team for updating inference related sections

Reply:
removed and updated